data/saas/config/twilio_sms_config.yml

```
@@ -0,0 +1,94 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Twilio SMS
+  type: twilio_sms
+  description: A sample schema representing the Twilio SMS integration for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/twilio-sms
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      label: Domain
+      default_value: api.twilio.com
+      description: Your Twilio SMS URL
+    - name: twilio_account_sid
+      label: Account SID
+      description: Your Account SID from the Twilio console, from the landing page select Account then API keys and tokens
+      sensitive: True
+    - name: twilio_auth_token
+      label: Auth token
+      description: Your auth token from the Twilio console, from the landing page select Account then API keys and tokens
+      sensitive: True
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: basic
+      configuration:
+        username: <twilio_account_sid>
+        password: <twilio_auth_token>
+
+  test_request:
+    method: GET
+    path: /v1/Users
+    client_config:
+      protocol: https
+      host: conversations.twilio.com
+      authentication:
+        strategy: basic
+        configuration:
+          username: <twilio_account_sid>
+          password: <twilio_auth_token>
+    query_params:
+      - name: PageSize
+        value: "1"
+
+  endpoints:
+    - name: user
+      requests:
+        read:
+          method: GET
+          path: /2010-04-01/Accounts/<twilio_account_sid>/Messages.json
+          query_params:
+            - name: From
+              value: <phone_number>
+          data_path: messages
+          pagination:
+            strategy: link
+            configuration:
+              source: body
+              path: next_page_uri
+          param_values:
+            - name: twilio_account_sid
+              connector_param: twilio_account_sid
+            - name: placeholder_phone
```

The `param_value` name is how you reference the value in the placeholders. As you have it now `< phone_number>` will be empty since you gave this value a name of `placeholder_phone`. This GET request is actually retrieving all messages!
```suggestion
            - name: phone_number
```